docs/tutorials/python/profiler.md

```diff
@@ -193,10 +193,10 @@ print(profiler.dumps())
 You can also dump the information collected by the profiler into a `json` file using the `profiler.dump()` function and view it in a browser.
 
 ```python
-profiler.dump()
+profiler.dump(finished=False)
```

Comment:
I don't think finished=False should be used in this tutorial. Most common usecase of an user using MXNet profiler is just running profiler once. We should just probably add a note, saying if the user wishes to run profiler in multiple sections of the code, then they should remember to set finished=False so profiler object is still not killed.

Reply:
If we do not use `finished = False` here, the code in the next section will error out in the tutorial nightly test.
Because you can only have one profiler per script, I think there might not be any workaround except for setting finished to False here

Reply:
Actually, another solution is to create a new tutorial for the next section (profiling custom operator section). This way, we have a brand new profiler